### ISSUE TITLE
Profile Detail Page FE A11y: Placeholder Image with Decorative Alt Text Throws Warnings

### ISSUE THREAD

**Issue by TaylorYeeBC** [BellevueCollege/bc-sitka-spruce-department-theme#475]
Bug related to Release v1.6.

The `alt=""` on the decorative Placeholder Image is semantically correct, but there are warnings thrown on the A11y tool and Editoria11y plugin. These should not be shown to Website Editors if they cannot be fixed.
The `a11y-decorative` CSS class should be added to the Image to have those warnings ignored by both the A11y tool and Editoria11y per Taija.

<img width="1914" height="980" alt="Image" src="https://github.com/user-attachments/assets/7ecf26d0-0142-437a-ac38-bf16ea0f06c7" />

**Comment by TaylorYeeBC** [comment]
Fixed with #474
